spec/component_guide/component_index_spec.rb

```diff
@@ -68,7 +69,7 @@
 @import 'govuk_publishing_components/components/tabs';
 @import 'govuk_publishing_components/components/title';"
 
-    expect(page).to have_selector(".component-doc-h2", text: "Gem components used by this app (13)")
+    expect(page).to have_selector(".component-doc-h2", text: "Gem components used by this app (15)")
```

Comment:
Will this expectation break when components are added to or removed from the gem?

Reply:
It will break if components are added or removed from the dummy gem. This does happen occasionally, but it's fairly easy to figure out.